_posts/2018-09-13-september-office-hours.md

```diff
@@ -0,0 +1,87 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-13
+image: /images/september-office-hours/team.jpg
```

Comment:
@alexellis should we restrict images to those with provable sources wrt licensing, CC/Public Domain etc?

Reply:
I'm not sure how to _prove_ that the image is free for use, but here's the link to where I downloaded it from: https://pixabay.com/en/team-team-building-success-computer-3373638/

Reply:
In this case showing the source has a CCA licence is the proof. The alternative is just random Google images sourced images where copyright is not clear.

Reply:
Is there somewhere that should be stored? In the post, or commit message?

Reply:
I think commit/PR message is a very good idea. Worth updating the contributing doc while you've this open to make future posters aware.